public/pages/CreateMonitor/containers/DefineMonitor/DefineMonitor.js

```diff
@@ -220,35 +259,51 @@ class DefineMonitor extends Component {
     const { dataTypes, response, performanceResponse } = this.state;
     const { index, searchType, timeField } = values;
     const isGraph = searchType === SEARCH_TYPE.GRAPH;
+    const isHTTP = searchType === SEARCH_TYPE.HTTP;
+    // Definition of when the "run" button should be disabled for HTTP type.
+    const disableHTTP =
+      (isHTTP && (values.http.urlType === URL_TYPE.FULL_URL && !values.http.url)) ||
+      (values.http.urlType === URL_TYPE.ATTRIBUTE_URL && !values.http.host);
```

Comment:
Do we require other settings in the ATTRIBUTE_URL case?

Reply:
I don't think so. Scheme will always be either http or https so I think having host would be enough.